data/datasets/tlcv2.0_oa/loading_script.py

```diff
@@ -0,0 +1,5 @@
+from datasets import load_dataset
```

Comment:
We don't need this file anymore, it's automatically done via `__init__.py` in the parent dir 🙂